kythe/proto/storage.proto

```diff
@@ -178,3 +178,21 @@ message ShardRequest {
   int64 index = 1;
   int64 shards = 2;
 }
+
+// KzipInfo contains a summary of the contents of a kzip. It provides a
+// breakdown of files and units by corpus and lanugage.
+message KzipInfo {
```

Comment:
Probably better next to the compilation unit messages in `analysis.proto`

Reply:
Done.